### DISCUSSION TITLE
large_image:Cannot use memcached for caching

### DISCUSSION THREAD

**Discussion by Flawks** [banesullivan/localtileserver#163] [Q&A] [answered]
What does  in the output 

> INFO:large_image:Cannot use memcached for caching.
> INFO:large_image:Using python for large_image caching

of the command `client = TileClient('image.tif')` mean?

I'm using Google Colab and here is the code:

```
!pip install localtileserver
!pip install ipyleaflet

from localtileserver import get_leaflet_tile_layer, TileClient
from ipyleaflet import Map

client = TileClient('image.tif')
t = get_leaflet_tile_layer(client)

m = Map(center=client.center(), zoom=client.default_zoom)
m.add_layer(t)
m
```

**Comment by banesullivan** [comment]
This is just a logging message from the large-image package which is designed to be used with a caching layer. We don't need or want this in `localtileserver`. Nothing to worry about.

**Reply by Flawks** [reply]
Does it affect the image display? I'm asking because I have troubles displaying the raster and looking for a possible cause.

**Reply by banesullivan** [reply, marked as answer]
No. This should have no effect
